package/.eslintrc.cjs

```diff
@@ -0,0 +1,61 @@
+// eslint-disable-next-line no-undef
+module.exports = {
+    env: {
+        browser: true,
+        es2021: true
+    },
+    extends: [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    overrides: [
+    ],
+    parser: "@typescript-eslint/parser",
+    parserOptions: {
+        ecmaVersion: "latest",
+        sourceType: "module"
+    },
+    plugins: [
+        "@typescript-eslint"
+    ],
+    ignorePatterns: [
+        "**/*.js",
+        "**/*.d.ts"
```

Comment:
Why ignoring this files?

Reply:
To stop eslint from linting compiled JS files from TS files

Reply:
ok, accepting that, but maybe just ignore the dist folder. Is more understandable.